docs/troubleshooting.md

```diff
@@ -35,3 +36,28 @@ using Minishift.
   systemctl start virtlogd
   systemctl enable virtlogd
   ```
+
+
+## XHYVE drivers
+
+- Minishift failed to start because of ```Error: could not create vmnet interface, permission denied or no entitlement```.
+
+    The problem is likely to be that the xhyve driver is not able to clean up vmnet when a VM is removed.
+
+    Workaround:
+        vmnet.framework decides the IP based on following files:
+
+            /var/db/dhcpd_leases
+            /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist
+
+    Reset `minishift` specific IP database and make sure you remove `minishift` entry section from `dhcpd_leases` file. Finally, reboot your system.
+
+            {
+                ip_address=192.168.64.2
+                hw_address=1,2:51:8:22:87:a6
+                identifier=1,2:51:8:22:87:a6
+                lease=0x585e6e70
+                name=minishift
+            }
+
+    You can do a complete reset IP database by removing both files manually but this is very **risky**.
```

Comment:
You can do a complete reset of the IP database by removing both the files

I think it would be better to add a  **Note** or **Warning** before the sentence.